.github/policies/issues.triage.yml

```diff
@@ -1,26 +1,139 @@
+# This file is generated by the sync-labels script DO NOT EDIT manually
 id: issues.triage
 name: New Issue Assign labels
 description: Assign labels to new issues
-owner:
 resource: repository
 disabled: false
-where:
 configuration:
   resourceManagementConfiguration:
-    scheduledSearches: []
     eventResponderTasks:
       - description: Adds `needs-triage` label for new unassigned issues
         if:
           - payloadType: Issues
           - isAction:
               action: Opened
-          - and:
-              - not:
-                  and:
-                    - isAssignedToSomeone
+          - not:
+              and:
+                - isAssignedToSomeone
         then:
           - addLabel:
               label: needs-triage
-
-onFailure:
-onSuccess:
+      - description: Remove `needs-triage` label when an area label is added
+        if:
+          - payloadType: Issues
+          - hasLabel:
+              label: needs-triage
+          - isOpen
+          - or:
+              - labelAdded:
```

Comment:
any chance we can define this in a common file so its not duplicated?

Reply:
Spoke offline these are generated files so they are using the common definition.  I suggest we rename to indicate it is generated to make it easier to know.